Sources/StreamChat/Workers/Background/MessageSender.swift

```diff
@@ -56,6 +56,9 @@ class MessageSender: Worker {
                 if let changes = self?.observer.items.map({ ListChange.insert($0, index: .init(item: 0, section: 0)) }) {
                     self?.handleChanges(changes: changes)
                 }
+                self?.database.write {
+                    $0.rescueMessagesStuckInSending()
+                }
```

Comment:
nit: Aren't we missing a test to verify that we rescue the messages when the session starts?